.devcontainer/Dockerfile

```diff
@@ -0,0 +1,91 @@
+FROM ubuntu:24.04
```

Comment:
I was hoping to use the statick docker image as a basis, but the last published one was too old, and based on ubuntu 20.04.

So I  opted to create a new one.